packages/samples/react/src/components/input-checkbox/partials/variants.tsx

```diff
@@ -25,6 +25,7 @@ export const InputCheckboxVariant: FC<Props> = ({ variant }) => {
 				_variant={variant}
 				_label="Nicht ausgewählt"
 				_value={false}
+				class="block"
```

Comment:
Das sollte die Komponente selber machen? :host ?

Die anderen Eingabefelder sind doch auch Block?

Reply:
Hintergrund war: die EU-Themes setzen grundsätzlich alle Elemente auf `display: inline-block`. Ich nehme an, das war eine gewollte/bewusste Entscheidung. In der Demo sah das teilweise nur nicht gut aus, deshalb die `block`-Klasse an den Stellen.